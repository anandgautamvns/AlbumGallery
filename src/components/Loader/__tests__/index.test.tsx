import {render} from '@testing-library/react-native';
import React from 'react';
import Loading from '..';

describe('Loading Component', () => {
  // it('should render the ActivityIndicator', () => {
  //   const {getByTestId} = render(<Loading />);
  //   const activityIndicator = getByTestId('activity-indicator');
  //   expect(activityIndicator).toBeTruthy();
  //   expect(activityIndicator.props.size).toBe('large');
  //   expect(activityIndicator.props.color).toBe('#0000ff');
  // });

  it('should render the loading text', () => {
    const {getByText} = render(<Loading />);
    expect(getByText('Loading...')).toBeTruthy();
  });

  it('should match the snapshot', () => {
    const {toJSON} = render(<Loading />);
    expect(toJSON()).toMatchSnapshot();
  });
});
