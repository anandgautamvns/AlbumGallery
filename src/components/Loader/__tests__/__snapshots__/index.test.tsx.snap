// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should match the snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    testID="activity-indicator"
  />
  <Text>
    Loading...
  </Text>
</View>
`;
